src/js/tabs/osd.js

```diff
@@ -300,6 +300,44 @@ OSD.initData = function() {
         osd_profiles: {},
     };
 };
+
+OSD.initVirtualMode = function() {
+    OSD.data.video_system = 1;
```

Comment:
Having this separate bit of initialisation out here is a bit of a pain. In reality osd.js should be refactored to not contain all the bits that don't belong there.
For now, how about the OSD data is initialised in `generateVirtualConfig()`, and then copied / linked into `osd.js` by an `importVirtualOsdConfig()` or suchlike?